File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int ans=0;
        for (auto e :nums){
            ans=ans^e;
        }
        int n= nums.size();
        for (int i=0; i<=n; i++){
            ans^=i;
        }

        return ans;
    }    

    
};

File: 268-missing-number/missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n= len (nums)
        s=0
        for i in nums:
            s+=i
        return ((n*(n+1))/2) -s
        